core-feature-pack/galleon-feature-pack/pom.xml

```diff
@@ -233,6 +233,8 @@
                             <fork-embedded>${galleon.fork.embedded}</fork-embedded>
                             <generate-channel-manifest>true</generate-channel-manifest>
                             <minimum-stability>experimental</minimum-stability>
+                            <config-stability>community</config-stability>
+                            <package-stability>experimental</package-stability>
```

Comment:
@jfdenise In a future iteration these 3 flags seem like things that should be part of wildfly-feature-back-build.xml.

The mojo options can remain as overrides (and for compatibility), but the other file seems like the right place for configuring the FP content, as opposed to the tool.

Reply:
@bstansberry , thank-you I logged https://issues.redhat.com/browse/WFGP-275 to track it.